.github/workflows/ci.yml

```diff
@@ -47,28 +43,22 @@ jobs:
       image: 'erlang:${{ matrix.otp_vsn }}'
     env:
       LATEST_OTP_RELEASE: 23.1
+      RUNNING_ON_CI: true
```

Comment:
There's already a CI=true variable set in ~all CIs. Did you try it?

Reply:
I didn't. What "CIs" are you talking about? Do you mean GitHub Actions itself declares it?

Reply:
All of them, including GA https://docs.github.com/en/free-pro-team@latest/actions/reference/environment-variables#default-environment-variables

Basically if there's a CI out there that doesn't set CI=true on its own, it's broken. Don't trust them. Amateurs 🙂

Reply:
6776065